packages/components/src/InputGroup/InputGroup.mdx

```diff
@@ -0,0 +1,55 @@
+---
+name: InputGroup
+menu: Components
+route: /components/input-group
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { InputGroup } from ".";
+import { InputText } from "../InputText";
+import { InputTime } from "../InputTime";
+import { CivilTime } from "@std-proposal/temporal";
+
+# InputGroup
+
+<ComponentStatus stage="rc" responsive="no" accessible="yes" />
+
+Group a different kinds fields to show a visual relationship. If a number of
+form fields once submitted make up a group of content (ie. an address, or first
+and last name becoming a full name) they are great candidates for an input
+group.
+
+The default layout is for fields to flow in a vertical direction, one above the
+next. Use the `flowDirection` prop to specify when you want fields to flow in a
+horizontal direction.
+
+**Warning** - Nested `InputGroup` components need to be of
+`flowDirection="horizontal"` to render properly. If you don't follow this, the
+group will not render and en error will appear in the browser console.
```

Comment:
```suggestion
> **Note:** Nested `InputGroup` components need to be of
> `flowDirection="horizontal"`. This ensures that we avoid layout issues
> cased by nesting. Any non-horizontal nested groups will log an error to
> the console and not be rendered
```

Quote and note might be a better / more consistent look for this?

Also played with the wording a bit…